sdk/include/xdk/winnt_old.h

```diff
@@ -952,9 +952,6 @@
 #define DLL_PROCESS_ATTACH    1
 #define DLL_THREAD_ATTACH    2
 #define DLL_THREAD_DETACH    3
-#ifdef __WINESRC__
-#define DLL_WINE_PREATTACH    8 /* Never called, but defined for compatibility with Wine source */
-#endif
 #define TAPE_ABSOLUTE_POSITION 0
```

Comment:
```suggestion

#define TAPE_ABSOLUTE_POSITION 0
```
Taking the opportunity to separate the group of defines here (DLL_PROCESS/THREAD_xxx and TAPE_xxx are unrelated).